meerk40t/gui/propertypanels/imageproperty.py

```diff
@@ -956,7 +995,10 @@ def generate_preview(self):
 
     @staticmethod
     def accepts(node):
-        if node.type == "elem image":
+        # Changing the staticmethod into a regular method will cause a crash, so therefore this circumvention
+        # Not the nicest thing in the world, as we need to instantiate the class once to reset the status flag
+        global HAS_VECTOR_ENGINE
```

Comment:
This is only needed if we're going to set that value. It's unneeded if we're just checking its value.